BUILD_MESON

```diff
@@ -122,20 +129,37 @@ Enable "unit-tests" option to run the test suite:
      
 Unlike Dinit's Make build system, We don't enable -fsanitize by default so if you need that use this option in meson setup step:
 
-     meson setup -Db_sanitizes='address,undefined' dirbuild/
+     meson setup -Db_sanitizes='address,undefined' builddir/
 
 Enable "igr-tests" to run the integration tests:
 
-     meson setup -Digr-tests=true dirbuild/
+     meson setup -Digr-tests=true builddir/
 
 (The integration tests are more fragile than the unit tests, but give a better indication that
 Dinit will actually work correctly on your system).
 
-Finally switch current directory to Your builddir (eg dirbuild), compile Dinit with meson compile and run tests via:
+Finally compile Dinit with "meson compile -C builddir" and run tests via:
+
+     meson test -C builddir
```

Comment:
Does this work for "older versions" of meson?

Reply:
Seems like yes, meson docs said:
> Legacy notes
If meson test does not work for you, you likely have a old version of Meson. In that case you should call `mesontest` instead. If `mesontest` doesn't work either you have a very old version prior to 0.37.0 and should upgrade.

I think 0.47.0 and later, have `meson test`. I figure it out.

Reply:
It may be easiest simply to require a newer version of meson, rather than try to document all the variations that may exist for older versions. But it's up to you.

Reply:
> It may be easiest simply to require a newer version of meson, rather than try to document all the variations that may exist for older versions. But it's up to you.

I prefer don't break compatibility with older mesons for now.

Reply:
Seems like `meson test` is available with meson 0.47.0 or later.